### PR TITLE
Ci: email validation

### PR DESCRIPTION
**Pull request by PedroChaparro**
Fix #62 CI errors.

Includes: 

- [x] Change the env variables names to follow the EMAIL_X convention. 
- [x] Add the missing env variables to the integration workflow. 
- [x] Create repository secret to securely save the email and secret key.

### CODE REVIEW

**Comment by PedroChaparro** [comment]
**Please,** wait for @SilviaPabon approval.

**Review by SilviaPabon — APPROVED** [review]

**Pull request merged**
